toolkit/analysis_and_visualization/classification/assets/scripts/app.js

```diff
@@ -0,0 +1,572 @@
+            }
+
+            if (!compare(itemValue, filters[key][1], filters[key][0])) {
+                allTrue = false;
+            } else {
+                anyTrue = true;
+            }
+
+
+        }
+        if (notOp === true) {
+            anyTrue = !anyTrue;
+            allTrue = !allTrue;
+        }
+        if (combinationOp == "or") {
+            return anyTrue;
+        } else if (combinationOp == "and") {
+            return allTrue;
+        }
+
+        return true;
+
+    });
+
+    if (allEmpty) {
+        var perPage = 50;
+        var pageNumber = 1;
+        imageSummaryFiltered = JSON.parse(JSON.stringify(imageSummary));
+        var showFrom = perPage * (pageNumber - 1);
+        var showTo = showFrom + perPage;
+        loadImageResults(imageSummaryFiltered.slice(showFrom, showTo));
+
+    } else {
+        loadImageResults(imageSummaryFiltered);
+    }
+    updateItems();
+}
+
+
+
+function loadImageResults(imageSummaryLocal) {
+    var myTableBody = $("#image-results-table").find("tbody");
+    myTableBody.empty();
+    var count = 0;
+    imageSummaryLocal.forEach(function (im) {
+
+        if (count > 40)
+            return;
+
+        count++;
+
+        var row = $("<tr>");
+
+        // var id = im.imgName;
+
+        var matchClass = im.match > 0 ? "color-green" : "color-red";
+        // var imgLink = $('a').attr('href', im.filePath).attr('target', '_blank').text(im.imageName);
+
+        var imgHtml = "<img onclick=\"showModal(event)\" src=\"" +
+            im.filePath +
+            "\" data-alt=\"<b>GROUND TRUTH:</b>" +
+            im.gtText + "<br><b>CLASSIFIED AS:</b>" +
+            im.labelTexts[0] + "\" width=\"30\" height=\"30\">";
+        var imgLinkHtml = "<a href=\"" + im.filePath + "\" target=\"_blank\">" + im.imageName + "</a>";
+
+        row.append($("<td>").attr("class", "imgcell").html(imgHtml));
+        row.append($("<td>").html(imgLinkHtml));
+        row.append($("<td>").attr("class", "imgcell left-align").text(im.gtText));
+        row.append($("<td>").attr("class", "imgcell").text(im.gt));
+
+        for (var i = 0; i < 5; i++) {
+            row.append($("<td>").attr("class", "imgcell").text(im.labels[i]));
+        }
+        row.append($("<td>").attr("class", "imgcell bold " + matchClass).text(im.match));
+
+        for (i = 0; i < 5; i++) {
+            row.append($("<td>").attr("class", "imgcell left-align").text(im.labelTexts[i]));
+        }
+
+        for (i = 0; i < 5; i++) {
+            row.append($("<td>").attr("class", "imgcell").text(im.probs[i].toFixed(4)));
+        }
+        myTableBody.append(row);
+    });
+
+
+}
+
+// eslint-disable-next-line no-unused-vars
+function showModal(event) {
+    var modal = document.getElementById("myModal");
+    var modalImg = document.getElementById("img01");
+    var modalCaption = document.getElementById("caption");
+    modal.style.display = "block";
+    modalCaption.innerHTML = event.target.dataset.alt;
```

Comment:
Codacy found an issue: [Unsafe assignment to innerHTML](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6397592)